pkg/version/version.go

```diff
@@ -0,0 +1,41 @@
+package version
+
+import (
+	"github.com/openshift/cluster-samples-operator/pkg/apis/samplesoperator/v1alpha1"
+	"k8s.io/apimachinery/pkg/version"
+)
+
+var (
+	// commitFromGit is a constant representing the source version that
+	// generated this build. It should be set during build via -ldflags.
+	commitFromGit string
+	// versionFromGit is a constant representing the version tag that
+	// generated this build. It should be set during build via -ldflags.
+	versionFromGit string
+	// major version
+	majorFromGit string
+	// minor version
+	minorFromGit string
```

Comment:
should have a patch field too?

Reply:
https://github.com/openshift/origin/blob/master/pkg/version/version.go and of the other version.go under origin/pkg didn't have a patch field